irc/varys/varys.go

```diff
@@ -43,6 +43,10 @@ type Client interface {
 
 	// SendRaw supports a blank uid to send to all connections.
 	SendRaw(uid string, params InterpolationParams, messages ...string) error
+	// GetNick gets the current connection's nick
+	GetNick(uid string, nick *GetNickParams) error
+	// Connected returns the status of the current connection
+	Connected(uid string, connected *ConnectedParams) error
```

Comment:
```suggestion
	// GetNick gets the current connection's nick
	GetNick(uid string) (string, error)
	// Connected returns the status of the current connection
	Connected(uid string) (bool, error)
```

The `Client` interface should be the simplified API — the `Varys` interface needs to match the requirements of the net/rpc library: https://golang.org/pkg/net/rpc/?m=all#pkg-overview